src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -391,6 +392,15 @@ private void OnHostInitializing(object sender, EventArgs e)
             IsHostHealthy(throwWhenUnhealthy: true);
         }
 
+        /// <summary>
+        /// Called after the host has been fully initialized, but before it
+        /// has been started.
+        /// </summary>
+        private void OnHostInitialized(object sender, EventArgs e)
+        {
+            State = ScriptHostState.Initialized;
```

Comment:
We never set Initialized previously? I seem to recall you mentioning this a while back... (maybe it was in the context of fixing this in v1).